pkg/roachprod/cloud/gc.go

```diff
@@ -421,10 +445,18 @@ func GCClusters(l *logger.Logger, cloud *Cloud, dryrun bool) error {
 		if err := destroyResource(dryrun, func() error {
 			return DestroyCluster(l, c)
 		}); err == nil {
-			destroyedClusters = append(destroyedClusters, resourceDescription{
-				Description:      fmt.Sprintf("%s (expiration: %s)", c.Name, c.ExpiresAt().String()),
-				SlackDescription: fmt.Sprintf("`%s` (expiration: %s)", c.Name, slackClusterExpirationDate(c)),
-			})
+			if !c.IsEmptyCluster() {
+				destroyedClusters = append(destroyedClusters, resourceDescription{
+					Description:      fmt.Sprintf("%s (cloud(s): %s, expiration: %s)", c.Name, strings.Join(c.Clouds(), ","), c.GCAt().String()),
+					SlackDescription: fmt.Sprintf("`%s` (cloud(s): `%s`, *expiration*: %s)", c.Name, strings.Join(c.Clouds(), ","), slackClusterExpirationDate(c)),
```

Comment:
Nit: for multi-cloud clusters, this will render as `gce, azure` instead, arguably nicer, `gce`, `azure`.

Reply:
Yep, I can make it prettier :)